integrationExamples/gpt/x-domain/creative.js

```diff
@@ -0,0 +1,56 @@
+function renderAd(ev) {
+    var key = ev.message ? 'message' : 'data';
+    var adObject = {};
+    try {
+      adObject = JSON.parse(ev[key]);
+    } catch (e) {
+      return;
+    }
+
+    if (adObject.ad || adObject.adUrl) {
+      var height = adObject.height;
+      var width = adObject.width;
+      var url = adObject.adUrl;
+      var ad = adObject.ad;
+      var doc = window.document;
+
+      // if (doc === document || adObject.mediaType === 'video') {
```

Comment:
Yep. This code is running in the x-domain iframe, so `doc === document`. Also `utils` module is not available so need to solve for a local `console.log`.